source/modules/models/transfer.md.erb

```diff
@@ -3,7 +3,17 @@
 Information regarding the transfer of purchased [InspetorItems](#inspetoritem) (*e.g. tickets*) within your platform.
 
 ### Properties
-
+```JSON
+{
+    "transfer_id":             "43262",
+    "sale_id":                 "12345",
```

Comment:
So, in other places, like the sale file above, IDs are integers, not strings. Is this behavior really supposed to be different between endpoints?

Reply:
yeah, don't look at the models files for now--I didn't update them since they are not part of our current documentation. (These should all be strings)